spring-integration-rsocket/src/main/java/org/springframework/integration/rsocket/ServerRSocketMessageHandler.java

```diff
@@ -60,15 +61,16 @@
 public class ServerRSocketMessageHandler extends IntegrationRSocketMessageHandler
 		implements ApplicationEventPublisherAware {
 
+	@SuppressWarnings("NullAway") // Reflection
 	private static final Method HANDLE_CONNECTION_SETUP_METHOD =
 			ReflectionUtils.findMethod(ServerRSocketMessageHandler.class, "handleConnectionSetup", Message.class);
 
-	private final Map<Object, RSocketRequester> clientRSocketRequesters = new HashMap<>();
+	private final Map<Object, @Nullable RSocketRequester> clientRSocketRequesters = new HashMap<>();
```

Comment:
After adding an assert on the `rsocketRequester` from headers, we won't need this one here.